### ISSUE TITLE
Internal denoising doesn't work when 'render threads' is set to '0'

### ISSUE THREAD

**Issue by jdent02** [bsavery/PRMan-for-Blender#285]
I'm working on a fix for this tonight but I just wanted to get it on the books in case anybody else runs across this. Long story short the denoise command line expects an actual number for rendering threads.  -1 works but '0' causes the denoiser to not launch, triggering an error

**Comment by jdent02** [comment]
FYI I'm planning to put in code that will covert the setting to the proper thread count if the user sets it to '0' in the exporter.

**Comment by bsavery** [comment]
Yep oh sorry I had a similar conversation with ryan a few months ago

On Saturday, June 18, 2016, Jon D <notifications@github.com> wrote:

> I'm working on a fix for this tonight but I just wanted to get it on the
> books in case anybody else runs across this. Long story short the denoise
> command line expects an actual number for rendering threads. -1 works but
> '0' causes the denoiser to not launch, triggering an error
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/bsavery/PRMan-for-Blender/issues/285>, or mute the
> thread
> <https://github.com/notifications/unsubscribe/ABzffNUDtP-keaTeEjfm4RrkfGvqxO4iks5qNHNpgaJpZM4I5DES>
> .
>


-- 
brian.savery@gmail.com
508-274-8700
